SimplenoteUITests/NoteEditor.swift

```diff
@@ -177,6 +185,13 @@ class NoteEditor {
         return matches
     }
 
+    class func openHistory() {
+        app.buttons[UID.Button.noteEditorMenu].tap()
+        let historyButton = app.tables.staticTexts[UID.Text.noteEditorOptionsHistory]
+        guard historyButton.waitForExistence(timeout: minLoadTimeout) else {return}
```

Comment:
Just in order to keep it along with the existing styling in the workspace:

```suggestion
        guard historyButton.waitForExistence(timeout: minLoadTimeout) else { return }
```